packages/eslint-plugin/docs/rules/consistent-type-exports.md

```diff
@@ -13,6 +13,22 @@ This rule aims to standardize the use of type exports style across a codebase.
 
 Given a class `Button`, and an interface `ButtonProps`, examples of code:
 
+```ts
+interface ButtonProps {
+  onClick: () => void;
+}
+class Button implements ButtonProps {
+  onClick() {
+    console.log('button!');
+  }
+}
+export { Button };
+export type { ButtonProps };
+
+//invalid example
+export { Button, ButtonProps };
```

Comment:
Please refer to how other rule docs are written. You should use two tabs, one incorrect and one correct.

Reply:
@Josh-Cena Should I write more explanations about the wrong example?